shotover-proxy/tests/cassandra_int_tests/protect.rs

```diff
@@ -13,77 +13,154 @@ pub struct Protected {
     _kek_id: String,
 }
 
-pub async fn test(shotover_session: &CassandraConnection, direct_session: &CassandraConnection) {
-    run_query(
-        shotover_session,
-        "CREATE KEYSPACE test_protect_keyspace WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };"
-    ).await;
-
-    run_query(
-        shotover_session,
-        "CREATE TABLE test_protect_keyspace.test_table (pk varchar PRIMARY KEY, cluster varchar, col1 blob, col2 int, col3 boolean);"
-    ).await;
-
-    run_query(
-        shotover_session,
-        "INSERT INTO test_protect_keyspace.test_table (pk, cluster, col1, col2, col3) VALUES ('pk1', 'cluster', 'I am gonna get encrypted!!', 0, true);"
-    ).await;
-
-    shotover_session.execute_batch(vec![
-        "INSERT INTO test_protect_keyspace.test_table (pk, cluster, col1, col2, col3) VALUES ('pk2', 'cluster', 'encrypted2', 1, true)".into(),
-        "INSERT INTO test_protect_keyspace.test_table (pk, cluster, col1, col2, col3) VALUES ('pk3', 'cluster', 'encrypted3', 2, false)".into()
-    ]).await;
-
-    let insert_statement = "BEGIN BATCH
-    INSERT INTO test_protect_keyspace.test_table (pk, cluster, col1, col2, col3) VALUES ('pk4', 'cluster', 'encrypted4', 3, true);
-    INSERT INTO test_protect_keyspace.test_table (pk, cluster, col1, col2, col3) VALUES ('pk5', 'cluster', 'encrypted5', 4, false);
-    APPLY BATCH;";
-    run_query(shotover_session, insert_statement).await;
-
+pub async fn select_all(shotover_session: &CassandraConnection) {
     // assert that data is decrypted by shotover
     assert_query_result(
```

Comment:
This large assert seems to be identical to the next one?
Should we run in a loop or move into a function to avoid duplicating it?